scripts/fetch/dv_data.R

```diff
@@ -6,6 +6,7 @@ fetch.dv_data <- function(viz = getContentInfo(viz.id = "NM_data")){
   site.id <- viz[["siteID"]]
   
   dailyData <- readNWISdv(site.id, "00060","","")
+  dailyData <- renameNWISColumns(dailyData)
```

Comment:
ah yeah, I can see that. Only thing is that adds 2 additional places I need to load dataRetrieval. At the moment, these functions don't need that. But I guess it's already a dependency of this viz, so it doesn't matter